dashboard/config/scripts_json/AlgebraA.script_json

```diff
@@ -20,7 +20,8 @@
       "is_course": true,
       "is_migrated": true,
       "use_legacy_lesson_plans": true,
-      "version_year": "unversioned"
+      "version_year": "unversioned",
+      "is_deprecated": true
```

Comment:
For my understanding - is this a flag that content editors can flip for future deprecations, or will this require an engineer?

Reply:
This will require an engineer. We have a flag content editors can flip (`published_state`), but we aren't ready to use it yet.